src/lamplib/src/genny/tasks/preprocess.py

```diff
@@ -220,6 +223,13 @@ def _parse_templates(self, templates):
                 template_node["TemplateName"], template_node["Config"], _ContextType.ActorTemplate
             )
 
+    def _parse_actors(self, actors):
+        actor_list = self._recursive_parse(actors)
```

Comment:
So..perhaps a hot-take but I actually dislike adding Actors the user didn't explicitly add since I (as a workload author) can't intuit about what's going on with this hidden step. 

Consider requiring the workload yamls to include this new Actor but perhaps with an easier syntax that this preprocesses into regular yaml?

```yaml
Actors:
  - Type: CollectPhaseMetrics
     # Barf on other keys and implicitly add Name and Threads
```

No biggie if this seems weird. Not a huge deal given this will show in `evaluate`.

Reply:
I'd personally steer away from `defaults.yml` for now. Although it's clever idea, I fear having more files to look at to understand things will breed confusion.

Whatever you decide here is fine by me. Easy to change depending on user feedback or future needs.